src/compiler/types.ts

```diff
@@ -4668,6 +4668,11 @@ namespace ts {
         readonly initializer: RequireOrImportCall;
     }
 
+    /* @internal */
+    export interface AccessedOrBareRequireVariableDeclaration extends VariableDeclaration {

```

Comment:
I think I prefer RequireVariableDeclarationOrExpression

Reply:
I'd agree (less easy to miss if you're doing completion and were about to use the old one, too).

Reply:
It’s not a “declaration or expression” though—it’s always a variable declaration, and the initializer of that variable declaration is always an expression. The thing that changes is whether that expression is a special call expression or a property access expression off that special call expression. I don’t like either of these names and don’t have a better suggestion ready, but I don’t understand what `RequireVariableDeclarationOrExpression` is supposed to mean.

Reply:
I think I'm going to go with `VariableDeclarationInitializedTo<T>` with `CallExpression` and `AccessExpression | CallExpression`.